README.md

```diff
@@ -23,6 +23,10 @@ A curated list of awesome Haskell people profiles you can support by sponsoring
 * [@tarleb — Albert Krewinkel](https://github.com/sponsors/tarleb)
 * [@vrom911 — Veronika Romashkina](https://github.com/sponsors/vrom911)
 
+## [OpenCollective](https://opencollective.com/)
+
+* [Fairy Tale - Artificial General Intelligence Solutions](https://github.com/fairy-tale-agi-solutions)
```

Comment:
Thanks for the addition! But it's not clear from the link how to sponsor your work. Could you provide a direct link to your open collective profile instead?

Reply:
Sure

Reply:
Done